src/mumble/CMakeLists.txt

```diff
@@ -138,6 +138,7 @@ set(MUMBLE_SOURCES
 	"ConnectDialogEdit.ui"
 	"ConnectDialog.h"
 	"ConnectDialog.ui"
+	"ConnectedQuitBehavior.h"
```

Comment:
```suggestion
	"ConnectedQuitBehavior.cpp"
	"ConnectedQuitBehavior.h"
```

Reply:
Do you always have a cpp file for a header even though there is no implementation in it?

Reply:
Oh wait - no we don't. I just remembered that there was a cpp file in your PR before but apparently that's gone now. In that case, never mind xD